<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelDialog</class>
 <widget class="QDialog" name="LabelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fit labels</string>
  </property>
  <property name="windowIcon">
   <iconset theme="format-text-italic">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="sourceLabel">
     <property name="text">
      <string>Source</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="sourceCombo"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="destonationLabel">
     <property name="text">
      <string>Destonation</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="destonationCombo"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="classLabel">
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="classCombo"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="distanceLabel">
     <property name="text">
      <string>Maximum label distance</string>
     </property>
     <property name="buddy">
      <cstring>distanceSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="distanceSpin">
     <property name="specialValueText">
      <string>Unlimited label distance</string>
     </property>
     <property name="suffix">
      <string> m</string>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="rotationLabel">
     <property name="text">
      <string>Rotation tolerance</string>
     </property>
     <property name="buddy">
      <cstring>rotationSpin</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDoubleSpinBox" name="rotationSpin">
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="infoCheck">
       <property name="text">
        <string>Check label text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>479</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="patternLabel">
     <property name="text">
      <string>Label Pattern</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="patternEdit">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>LabelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>LabelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>classCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LabelDialog</receiver>
   <slot>currentClassChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>currentClassChanged(int)</slot>
 </slots>
</ui>
